java/core/src/java/org/apache/orc/impl/RecordReaderUtils.java

```diff
@@ -481,26 +445,40 @@ static BufferChunk findSingleRead(BufferChunk first) {
    * @param zcr the zero copy shim
    * @param list the disk ranges within the file to read
    * @param doForceDirect allocate direct buffers
+   * @param minSeekSize the minimum gap to prefer seek vs read
+   * @param minSeekSizeTolerance allowed tolerance for extra bytes in memory as a result of
+   *                             minSeekSize
    */
   static void readDiskRanges(FSDataInputStream file,
```

Comment:
Do we need to add this method as public? Shall we add it as `private`?

Reply:
sure, made it private